plugins/woocommerce-admin/client/marketing/overview-multichannel/Channels/RegisteredChannelCardBody.scss

```diff
@@ -0,0 +1,47 @@
+.woocommerce-marketing-registered-channel-card-body {
+	.woocommerce-marketing-registered-channel-description {
+		display: flex;
+		gap: $gap-smaller;
+
+		&__separator::before {
+			content: '•';
+		}
+	}
+
+	.woocommerce-marketing-sync-status {
+		display: flex;
+		align-items: center;
+		gap: $gap-smallest;
+
+		&__failed {
+			color: $alert-red;
+			fill: $alert-red;
+		}
+
+		&__syncing {
+			color: #008a20;
+			fill: #008a20;
+		}
+
+		&__synced {
+			color: #008a20;
+			fill: #008a20;
+		}
```

Comment:
💅  Im unsure if there are some variables for these colors. Or we can use some variable defined colors for it.

Also, we can omit the duplication by using `&__syncing, &__synced `

Reply:
Good catch. Done in [`316aad8` (#36541)](https://github.com/woocommerce/woocommerce/pull/36541/commits/316aad8b2e460df1a6986ac67950c33b36988f7f).